### ISSUE TITLE
GTN CoFest (15th of November)

### ISSUE THREAD

**Issue by bebatut** [galaxyproject/training-material#1082]
![cofests](https://user-images.githubusercontent.com/1842467/46872640-fe9b1e00-ce34-11e8-8e4e-15d4a9a3cf9a.png)

We are organizing regular online CoFests (Collaboration/Contribution Fests) every 3 months on the 3rd Thursday for a day of the collaborative work on the training content.

**The next one will be on the 15th of November.**

We plan to focus on genome annotation training; but anyone who would like to contribute on any other topics is very welcome to join.

We will coordinate via [Hangout](https://hangouts.google.com/hangouts/_/calendar/Z2FsYXh5dW5pZnJlaWJ1cmdAZ21haWwuY29t.459bpoqjjo780nm56q267ba7i8?authuser=1) (drop-in channel we will keep open the whole day) and the [GTN Gitter channel](https://gitter.im/Galaxy-Training-Network/Lobby).

During the day, we will also have a call meeting with the GTN to discuss the recent changes and future directions (#1088)

---

*@bebatut and @shiltemann and @erasche will be online between 9am and 6pm (CEST) but people from other time zones should feel welcome to continue even before/after. We will keep an eye on Gitter and GitHub*

**Comment by bebatut** [comment]
We have many ideas for this CoFest, and participants can work on a wide range of topics depending on their interests:

- **Content**
  - **Add new topics, new tutorials**
  - [Training Handbook for instructors](https://github.com/galaxyproject/training-material/pull/1101)
  - Test existing tutorials and make improvements or [file issues](https://github.com/galaxyproject/training-material/issues)
    - [List of tutorials needing some love](https://github.com/galaxyproject/training-material/issues/807)
    - Improve the English there
  - Review [open pull requests](https://github.com/galaxyproject/training-material/pulls)

- **Global Infrastructure**
  - FAIRness evaluation of the training
  - Plan internationalisation and localisation support in order to make the training material available to users in their own language: [open PR](https://github.com/galaxyproject/training-material/pull/862) and [some discussion about long term plan](https://github.com/galaxyproject/training-material/issues/404)
  - [BioSchemas](http://bioschemas.org/) support
  - Complete integration with the [TeSS workflow annotator](https://tess.elixir-europe.org/workflows)

- **Technical support**
  - Workflow testing
  - Docker images
  - [Interactive tours](https://crs4.github.io/Galaxy4Developers/lectures/09.galaxy_interactive_tours/)

- **Community**
  - [More visibility for training events](https://github.com/galaxyproject/training-material/issues/1054)
  - Support for Instructors
  - [Instructors on the training website](https://github.com/galaxyproject/training-material/issues/1055)
  - [Share your training philosophy](https://github.com/galaxyproject/training-material/issues/1056) and tips to help other instructors

For those looking for ideas for things to work on this week, check out the [open issues and pull requests with the *cofest* label](https://github.com/galaxyproject/training-material/labels/CoFest).

**Comment by abretaud** [comment]
As I said to @bebatut, not sure how I will be able to participate, but here's a raw list of ideas for genome annotation (some easy, some not):

- I have a few small changes to do to the Maker tutorial after the first session we did on it. I also need to fix the MPI support in the Maker conda package (jobs are just too long without it)
- A tour for the Maker tutorial
- a JBrowse tutorial? loading multiple kind of data, playing with all the options (I guess @erasche already have some ideas for that!)
- some functional annotation stuff for eucaryotic species? in an ideal world I'd like a tutorial with blast, interproscan (conda packaging is a problem, and license too), blast2go or equivalent (license problem).
- orthology stuff with OrthoFinder? (the iuc tool needs some love if I remember well)
- some GGA tutorials? like "How can I refine the annotation produced by Maker (or Prokka etc) with Apollo from Galaxy?". Maybe also "How can I set up a public jbrowse/chado/tripal website from Galaxy"?
